dockerfiles/init/modules/compose/templates/docker-compose.yml.erb

```diff
@@ -23,16 +23,36 @@ che:
     - '/etc/group:/etc/group:ro'
     - '/etc/passwd:/etc/passwd:ro'
 <% end -%>
-  ports:
-    - '<%= scope.lookupvar('che::che_port') -%>:<%= scope.lookupvar('che::che_port') -%>'
 <% if scope.lookupvar('che::che_env') == 'development' -%>
+  ports:
+    - '8080'
     - '<%= scope.lookupvar('che::che_debug_port') -%>:<%= scope.lookupvar('che::che_debug_port') -%>'
 <% end -%>
   restart: always
+  labels:
+    traefik.che.frontend.backend: "che-server"
+    traefik.che.frontend.entryPoints: "http"
+    traefik.che.port: "8080"
+    traefik.che.frontend.rule: "PathPrefix:/"
   container_name: <%= ENV["CHE_CONTAINER_NAME"] %>
 <% if scope.lookupvar('che::che_user') != 'root' -%>
   user: <%= scope.lookupvar('che::che_user') -%>
 <% end -%>
 <% if ! @dns_resolvers.empty? -%>
 <%= "  dns:" + "\n" + @dns_resolvers.split(",").map { |val| "    - #{val}" }.join("\n") %>
 <% end -%>
+
+
+traefik:
+  image: florentbenoit/traefik
```

Comment:
yes sure, it's still a WiP
it was to allow me to use Traefik with my PR merged
when Traefik will release 1.3.x version I will switch in my PR